src/relayclient/RelayClient.ts

```diff
@@ -241,22 +243,25 @@ export class RelayClient {
       if ((transactionDetails.isSmartWalletDeploy ?? false)) {
         const estimated = await this.calculateSmartWalletDeployGas(transactionDetails)
         transactionDetails.gas = `0x${estimated.toString(16)}`
+        log.debug(`Relay Client - Estimated gas for deploying: ${transactionDetails.gas}`)
       } else {
         const estimated = await this.contractInteractor.estimateGas(this.getEstimateGasParams(transactionDetails))
         transactionDetails.gas = `0x${estimated.toString(16)}`
+        log.debug(`Relay Client - Estimated gas for relaying: ${transactionDetails.gas}`)
```

Comment:
this one and the one on line `246` could be removed and place after line `251` (after the `if` closes)